From abd74b7026050d0be702b46f8c0a693de54144fc Mon Sep 17 00:00:00 2001
From: UCHINO Satoshi <satoshi.uchino@toshiba.co.jp>
Date: Mon, 9 Jul 2012 10:23:55 +0900
Subject: [PATCH 032/859] LTTng: fix module name: lttng-relay.ko =>
 lttng-tracer.ko

It seems lttng-tools 2.0 expects lttng-tracer.ko instead of
lttng-relay.ko.

Signed-off-by: UCHINO Satoshi <satoshi.uchino@toshiba.co.jp>
Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
---
 drivers/staging/lttng/Makefile | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/staging/lttng/Makefile b/drivers/staging/lttng/Makefile
index 887e8a5..0f67d31 100644
--- a/drivers/staging/lttng/Makefile
+++ b/drivers/staging/lttng/Makefile
@@ -9,8 +9,8 @@ obj-m += lttng-ring-buffer-client-mmap-discard.o
 obj-m += lttng-ring-buffer-client-mmap-overwrite.o
 obj-m += lttng-ring-buffer-metadata-mmap-client.o
 
-obj-m += lttng-relay.o
-lttng-relay-objs :=  lttng-events.o lttng-abi.o \
+obj-m += lttng-tracer.o
+lttng-tracer-objs :=  lttng-events.o lttng-abi.o \
 			lttng-probes.o lttng-context.o \
 			lttng-context-pid.o lttng-context-procname.o \
 			lttng-context-prio.o lttng-context-nice.o \
@@ -23,11 +23,11 @@ obj-m += lttng-statedump.o
 lttng-statedump-objs := lttng-statedump-impl.o wrapper/irqdesc.o
 
 ifneq ($(CONFIG_HAVE_SYSCALL_TRACEPOINTS),)
-lttng-relay-objs += lttng-syscalls.o
+lttng-tracer-objs += lttng-syscalls.o
 endif
 
 ifneq ($(CONFIG_PERF_EVENTS),)
-lttng-relay-objs += $(shell \
+lttng-tracer-objs += $(shell \
 	if [ $(VERSION) -ge 3 \
 		-o \( $(VERSION) -eq 2 -a $(PATCHLEVEL) -ge 6 -a $(SUBLEVEL) -ge 33 \) ] ; then \
 		echo "lttng-context-perf-counters.o" ; fi;)
-- 
1.8.1.4

